### PR TITLE
Updating readme with link from cohort 3 video 

### PR DESCRIPTION
**Pull request by novica**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,8 +87,8 @@ Chapter 3 - Vectors
 Chapter 4 - Subsetting
 
 - Cohort 1: [Scott Nestler](https://twitter.com/ScottNestler) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort1/Chap4slides.html#1) | [video](https://www.youtube.com/watch?v=eLMpCc0t1cg))
-
-Chapter 5 - Control flow
```
Comment (tanho63) [inline README.md:92]: Hmm. Looks like you dropped this chapter five line

Comment (novica) [reply README.md:92]: I think I fixed that in the second commit.

File: README.md [hunk 2]
```diff
@@ -88,8 +87,10 @@ Chapter 3 - Vectors
 Chapter 4 - Subsetting
 
 - Cohort 1: [Scott Nestler](https://twitter.com/ScottNestler) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort1/Chap4slides.html#1) | [video](https://www.youtube.com/watch?v=eLMpCc0t1cg))
+- Cohort 2: ...
+- Cohort 3: ... [video](https://www.youtube.com/watch?v=CvvXkXiF3Ig)
```
Comment (tanho63) [inline README.md:91]: Just reviewing Shel's slides here, does it make sense to credit her and Alan? (I wasn't around to watch the presentation)

Comment (tanho63) [reply README.md:91]: Can also include links to slide (https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort3/chapter4.html#1)

Comment (novica) [reply README.md:91]: Yes. They worked together on the slides and presented together.

Comment (Shelmith-Kariuki) [reply README.md]: Thanks. I wanted to make another pull request to add the slides link to the readme, but I guess Tan you can do that for me? Or should I do it?

Comment (tanho63) [reply README.md]: I just got Novica to do it 😎 good job today!

**Pull request merged**